samples/bluetooth/central_hr/src/main.c

```diff
@@ -113,7 +113,7 @@ static bool eir_found(struct bt_data *data, void *user_data)
 
 		for (i = 0; i < data->data_len; i += sizeof(uint16_t)) {
 			struct bt_le_conn_param *param;
-			struct bt_uuid *uuid;
+			const struct bt_uuid *uuid;
```

Comment:
@Thalley  
> You haven't shared what the compiler warning is, which is what I am requesting :)

if not add `const` there, compiler will warning for that:

<img width="1057" alt="image" src="https://github.com/zephyrproject-rtos/zephyr/assets/26378606/766ce5b8-21d3-4f80-a0e7-b8b6f363ac3a">

Reply:
But, only warning, not build error.